examples/concertbot/README.md

```diff
@@ -0,0 +1,48 @@
+# Concertbot
+
+Example bot that contains only story data.
+
+## What’s inside this example?
+
+This example contains some training data and the main files needed to build an 
+assistant on your local machine. The `concertbot` consists of the following files:
+
+- **data/stories.md** contains training stories for Core model  
+- **actions.py** contains some custom actions
+- **domain.yml** contains the domain of the assistant  
+- **endpoints.yml** contains the webhook configuration for the custom actions 
+- **config.yml** contains the model configuration
```

Comment:
We should probably order these in the same order they are on the github repo i.e. folders then files, alphabetical